docs/developers-guide/driver-changelog.md

```diff
@@ -22,6 +22,12 @@ title: Driver interface changelog
 
 - Added the driver multi-methods `metabase.driver/schema-exists?` and `metabase.driver/create-schema-if-needed!` which should be implemented by drivers that support `:schemas` and `:transforms/table`.
 
+- Added `metabase.driver/type->database-type` multimethod that returns the database type for a given Metabase type (from the type hierarchy) as a HoneySQL spec. This method handles general Metabase base types.
+
+- Added `metabase.driver/insert-from-source!` multimethod that abstracts data insertion from various sources into existing tables. This multimethod dispatches on both the driver and the data source type (`:rows` or `:jsonl-file`). It allows drivers to optimize based on the data source type and returns the number of rows inserted. Default implementations are provided for both `:rows` and `:jsonl-file` source types.
+
+- Added `metabase.driver/string->val` multimethod to parse string values for insertion based on driver and column definition. Drivers should implement this when their insertion mechanism cannot handle string values directly and needs them converted to proper types.
```

Comment:
nit: line breaks would be nice, but I see other entries have forgotten this. It looks like other parts of the file wrap at around ~110 chars.